Shared Widgets/LockScreenCircularView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  LockScreenCircularView.swift
+//  St Jude
+//
+//  Created by Ben Cardy on 04/09/2022.
+//
+
+import SwiftUI
+import WidgetKit
+
+struct LockScreenCircularView: View {
+    
+    let campaign: TiltifyWidgetData
+    var shouldShowFullCurrencySymbol: Bool = false
+    var shouldShowGoalPercentage: Bool = false
+    
+    var body: some View {
+        ZStack {
+            if #available(iOSApplicationExtension 16.0, *) {
```

Comment:
No fall back if iOS 16 isn't available?

Reply:
This widget is only available on iOS 16 (the `.accessoryXXX` widget types), so the entire view could be wrapped in it really. It'll never get used on any other platform.